tests/metabase/management/test_populate_metabase_emplois.py

```diff
@@ -874,14 +874,16 @@ def test_populate_users():
         management.call_command("populate_metabase_emplois", mode="users")
 
     with connection.cursor() as cursor:
-        cursor.execute("SELECT * FROM utilisateurs ORDER BY id")
+        cursor.execute("SELECT * FROM utilisateurs_v0 ORDER BY id")
         rows = cursor.fetchall()
         assert rows == [
             (
                 pro_user.id,
                 pro_user.email,
                 "employer",
                 datetime.date(2023, 2, 1),
+                pro_user.first_name,
+                pro_user.last_name
```

Comment:
L'erreur du _make quality_ vient sûrement de là :
```suggestion
                pro_user.last_name,
```
Et le test va sûrement échouer car à mon avis il faut que les nouveaux champs soient avant la date.